server/test/repository/AccountRepositoryTest.java

```diff
@@ -45,6 +48,61 @@ public void listApplicants_empty() {
     assertThat(allApplicants).isEmpty();
   }
 
+  @Test
+  public void updateClientEmail_ThrowsEmailExistsException() {
+    setupAccountForUpdateTest();
+    AccountModel account = new AccountModel();
+    account.setEmailAddress("test1@test.com");
+    account.save();
+    assertThatThrownBy(() -> repo.updateClientEmail("test@test.com", account))
+        .isInstanceOf(EmailAddressExistsException.class);
+  }
+
+  @Test
+  public void updateClientNameTest_AllNonEmpty() {
+    ApplicantModel applicantUpdateTest = setupApplicantForUpdateTest();
+    repo.updateClientName("John", "James", "Dow", applicantUpdateTest);
+    assertThat(applicantUpdateTest.getApplicantData().getApplicantLastName().get())
+        .isEqualTo("Dow");
+    assertThat(applicantUpdateTest.getApplicantData().getApplicantMiddleName().get())
+        .isEqualTo("James");
+    assertThat(applicantUpdateTest.getApplicantData().getApplicantFirstName().get())
+        .isEqualTo("John");
+  }
+
+  @Test
+  public void updateClientNameTest_EmptyMiddleAndLastName() {
```

Comment:
question (blocking): Caitlin commented on this above, but should Last Name be required? It looks required in the UI.

Reply:
It is required but in the DB level, we see it as nullable field. I have also followed suite. Hence this test.